Userland/Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -566,4 +566,20 @@ void HTMLElement::did_receive_focus()
     browsing_context->set_cursor_position(DOM::Position::create(realm(), *this, 0));
 }
 
+String HTMLElement::access_key_label() const
+{
+    auto access_key = get_attribute(HTML::AttributeNames::accesskey);
+    if (!access_key.has_value() || access_key->is_empty())
+        return {};
+
+    StringBuilder access_key_label;
+    dbgln("FIXME: Implement HTMLElement::access_key_label() valid system key check");
+    // Loop through words in access_key to find first valid system key
+    dbgln("FIXME: Implement HTMLElement::access_key_label() get system modifier key");
+    // get system modifier key and append that instead of this line
```

Comment:
For steps that you don't want to implement in this change set, it's preferred to do something more like this:

```
// FIXME: N. Some verbatim text from the spec
// FIXME: N+1. Some other verbatim text from the spec
```

If the step that you're marking as FIXME has substeps, you can skip them if the parent step is FIXME'd.